controllers/jobs.js

```diff
@@ -73,7 +73,7 @@ const deleteJob = async (req, res) => {
     throw new NotFoundError(`No job with id ${jobId}`);
   }
 
-  res.status(StatusCodes.OK).send();
+  res.status(StatusCodes.OK).json({msg: "The entry was deleted."})
```

Comment:
I've seen this in another students' work, so I'm assuming the course tells you to do this but I'd advise against it. Receiving a success response from a DELETE request gives the client enough information to know the request is deleted, and they can then show whatever message they wish.

Reply:
The assignment wanted us to send a message with the response.